Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-alpine
+
+ENV PYTHOUNNBUFFERED 1
```

Comment:
There is a typo in the environment variable name. It should be `PYTHONUNBUFFERED` instead of `PYTHOUNNBUFFERED`.